README.md

```diff
@@ -1,46 +1,62 @@
 # Mina zkApp CLI
 
-The Mina zkApp CLI allows you to scaffold, write, test, & deploy zkApps
-("zero-knowledge apps") for [Mina Protocol](https://minaprotocol.com/) using
-recommended best practices. Apps are written using
-[SnarkyJS](https://docs.minaprotocol.com/en/zkapps/snarkyjs-reference), a
-TypeScript framework for writing zero-knowledge proof-based smart contracts,
-which is included by default in projects created using this CLI.
+The Mina zkApp CLI allows you to scaffold, write, test, and  deploy zkApps (zero-knowledge apps) for [Mina Protocol](https://minaprotocol.com/) using recommended best practices. 
 
-## Getting started
+Apps are written using [SnarkyJS](https://docs.minaprotocol.com/en/zkapps/snarkyjs-reference), a TypeScript framework for writing zero knowledge proof-based smart contracts. SnarkyJS is automatically included when you create a project using the Mina zkApp CLI.
 
-To get started, please read this README, followed by [Mina Protocol's zkApp
-docs](https://docs.minaprotocol.com/zkapps) for a step-by-step guide.
+## Getting Started
 
-## Dependencies
+To get started, read through this README file and the Mina Protocol [zkApp Developer](https://docs.minaprotocol.com/zkapps) docs. 
 
-You'll need the following installed to use the zkApp CLI:
+Build foundational knowledge by completing the guided steps in the [zkApp Developer Tutorials](https://docs.minaprotocol.com/zkapps/tutorials).
 
-- NodeJS 16+ (or 14 using `--experimental-wasm-threads`)
-- NPM 6+
-- Git 2+
+## Get Involved
```

Comment:
It will be goo to mention the CONTRIBUTING.md link in this section with brief description.
Other than that looks good, thank you.